### PR TITLE
[feat] Create Product page

### PR DESCRIPTION
**Pull request by Hamdrive**
Created products page of FueledUp™ store

Features:
1. Sidebar with various options to filter products
2. Large cards with information on products
3. Minimal responsiveness added for mobile (to be refactored later)

### CODE REVIEW

**Review by junaidshaikh-js — APPROVED** [review]
Good Work! 

Clean code, nice use of semantic elements.

File: css/products.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  padding: var(--size-3) var(--size-5);
+  border-radius: var(--radius-large);
+}
+
+.strike-color {
+  color: var(--btn-error-color);
+}
+
+.grid-cols-auto {
+  grid-template-columns: repeat(auto-fit, minmax(18rem, 18rem));
```
Comment (adith-a-danthi) [inline css/products.css]: You can probably skip `minmax` here since both `min` and `max` values are the same.

**Review by adith-a-danthi — COMMENTED** [review]
Looks Good! Clean and well-structured code.

File: css/products.css (hunk 1 above)
Comment (Hamdrive) [reply css/products.css]: Valid point, current implementation seems redundant
Thank you for pointing out,have updated code

**Pull request merged**